fv3gfs/util/utils.py

```diff
@@ -97,6 +97,14 @@ def device_synchronize():
         cp.cuda.runtime.deviceSynchronize()
 
 
+def flatten(array: np.ndarray) -> np.ndarray:
+    """Offers a common API for array flatten between numpy and cupy"""
+    if cp and isinstance(array, cp.ndarray):
+        return array.flatten()
+    else:
+        return array.flatten(order="C")
```

Comment:
This function should be equivalent to calling `array.flatten()`. numpy's [default value](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.flatten.html) for the "order" argument is "C". Can you delete this helper and use `array.flatten()` instead?

Reply:
Done.